docs/source/app-dev/bindings-ts/daml2js.rst

```diff
@@ -1,18 +1,12 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-DAML to JavaScript code generation
-##################################
+DAML/JavaScript Code Generator
```

Comment:
This sounds like you can generate either DAML or JavaScript. I find `to` to be easier to understand. Or just `JavaScript` codegentor.